OrbitService/Process.cpp

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "Process.h"
```

Comment:
would be nice to have commit moving Process separate from this commit - this way it is more visible what was changed.

Reply:
I will tackle that when Davide's comments are resolved.

Reply:
This is now in a separate commit. I'm going to make some more tests before I re-request review.